crates/net/discv4/src/lib.rs

```diff
@@ -1402,17 +1403,27 @@ impl Discv4Service {
             Entry::Vacant(_) => {
                 // received neighbours response without requesting it
                 trace!(target: "discv4", from=?remote_addr, "Received unsolicited Neighbours");
-                return
+                return;
             }
         };
 
+        // log the peers we discovered
+        trace!(target: "discv4",
+            target=format!("{:#?}", node_id),
+            peers_count=msg.nodes.len(),
+            peers=format!("[{:#}]", msg.nodes.iter()
+                .map(|node_rec| node_rec.id
+            ).format(", ")),
+            "discovered peers from Neighbours packet"
```

Comment:
we need to be a bit clear here because we didn't actually necessarily discover any new peers; maybe we already know them. let's just say we received them for now.

```suggestion
            "received peers from Neighbours packet"
```

Reply:
great. I'll make the correction.